back/src/api/authentication/changePassword/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+
+import { validatedExpressRequest } from '../../../core/utils/validatedExpressRequest';
+import { jwtMiddleware } from '../../../core/middlewares/jwt.middleware';
+import route from './route';
+import validator from './validator';
+
+export const changePassword = Router().post(
+  '/password/change',
+  jwtMiddleware({ isAgent: true }),
```

Comment:
Il vaut mieux ajouter le rate limiter pour cet endpoint :

```suggestion
export const changePassword = Router().post(
  '/password/change',
  rateLimiterMiddleware({ type: RateLimiterType.DEFAULT }),
  jwtMiddleware({ isAgent: true }),
```

ça permet de vérifier qu'un utilisateur ne fait pas plus de 10 tentatives toute les 10min

Plus tard, on devra ajouter un rate limiter spécific qui prends en compte le curent user dans le jwt aussi